### PR TITLE
[22] - our-story-section

### PR DESCRIPTION
**Pull request by Brandt-Bowling**
Tried to mimic layouts previously used by other sections. The image seems to be a placeholder from figma so we could revisit it when an actual image is provided.

image

image

### CODE REVIEW

File: frontend/src/components/page-about/index/our-story.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
+import * as React from "react"
+import hmiImage from "../../../images/about-our-story-greeters.png"
+import { HighlightedParagraph } from "../../shared/highlightedParagraph"
+
+const OurStory = () => {
+  return (
+    <div className="flex items-start gap-14">
```
Comment (thedomcab) [inline frontend/src/components/page-about/index/our-story.js:7]:
```suggestion
    <div className="flex items-start gap-[60px] px-0.5">
```

File: frontend/src/components/page-about/index/our-story.js [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+      <div className="basis-689/1180">
```
Comment (thedomcab) [inline frontend/src/components/page-about/index/our-story.js:8]:
```suggestion
      <div className="basis-[49.32%]">
```

File: frontend/src/components/page-about/index/our-story.js [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+            Although the composition of our church may continue to change, our
+            mission will always remain the same. <span>We desire to see people’s lives
+            transformed, redirected to Gods purposes, and set out to transform
+            the world for the glory of God.</span>
+          </p>
+        </HighlightedParagraph>
+      </div>
+
+      <div>
+        <img src={hmiImage} alt="Greeter" />
```
Comment (thedomcab) [inline frontend/src/components/page-about/index/our-story.js:42]:
```suggestion
        <img src={hmiImage} alt="Greeter"   className="mb-0" />
```
Based upon the mock, we should add text on top of the image.  
here is my suggestion for text
```
<div className="relative">
<img />
<div className="flex justify-center absolute top-0 bottom-0 left-0 right-0"> text</div>
</div>
```

Comment (Brandt-Bowling) [reply frontend/src/components/page-about/index/our-story.js]: I'm not sure what you mean by the last bit? Which text are you referring to from the mock?

Comment (thedomcab) [reply frontend/src/components/page-about/index/our-story.js]: The text in the image. from the the mock it shows text on top of the image.  So we easily change the text intstead redo the same image. 

image

Comment (Brandt-Bowling) [reply frontend/src/components/page-about/index/our-story.js]: Hey @thedomcab I checked with Phil, there's not going to be text over the image. The image itself is a placeholder and the text in the figma design was a note for himself to add a "real" one later

**Review by xumaple — APPROVED** [review]

**Review by xumaple — APPROVED** [review]

**Pull request merged**